tests/test_rio_merge.py

```diff
@@ -465,13 +462,13 @@ def test_merge_tiny_res_bounds(tiffs, runner):
 
     # Output should be
     # [[[120  90]
-    #   [40    0]]]
+    #   [60    0]]]
```

Comment:
The new compositing code has a sub-pixel shift compared to the previous boundless VRT version and the `40` raster (a-sw.tif) no longer contributes.